common/app/slices/CommercialContainer.scala

```diff
@@ -0,0 +1,13 @@
+package slices
+
+object CommercialContainer {
+  val all: Map[String, Container] = Map(
+    ("commercial/single-campaign", Commercial(SingleCampaign(FixedContainers.fixedSmallSlowIV))),
+    ("commercial/multi-campaign", Commercial(MultiCampaign(FixedContainers.fixedSmallSlowIV)))
+  )
+}
+
+sealed trait CommercialContainer
```

Comment:
The thing about `campaign` is that who knows what types of commercial containers we'll have to create in the future. `campaign` is what these two containers do right now, it does not describe what they are